### PR TITLE
Multilang: add missing Japanese translations

### PR DESCRIPTION
**Pull request by ambientocclusion**
**Description** 
Add missing Japanese translations.

**Verification** 
Translated by a native.

### CODE REVIEW

**Comment by sshane** [comment]
I added some missing translations on master, can you rebase and add them?

File: selfdrive/ui/translations/main_ja.ts [hunk 1]
```diff
@@ -70,7 +70,7 @@
     <message>
         <location line="+0"/>
         <source>leave blank for automatic configuration</source>
-        <translation>空白のままにして、自動設定にします</translation>
+        <translation>自動で設定するには、空白のままにしてください。</translation>
```
Comment (sshane) [inline selfdrive/ui/translations/main_ja.ts:73]: Do you mind opening a second PR with all these improvements to verify separately? That way it's easier to merge the additions!

Comment (ambientocclusion) [reply selfdrive/ui/translations/main_ja.ts]: Ok I will make another PR after this merge is accepted.

Comment (sshane) [reply selfdrive/ui/translations/main_ja.ts]: I have some requests when you open a PR, some translations are bit too long

**Comment by sshane** [comment]
@ambientocclusion do you mind doing some new unfinished translations?

**Comment by ambientocclusion** [comment]
@sshane 
I made new PR for new translations. 
https://github.com/commaai/openpilot/pull/26482

**Pull request merged**